### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by StarMindz**
Refractor all class components to functions using hooks

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (topeogunleye) [inline src/components/Calculator.js:1]: - [ ] Your calculator does not work after refactoring. Pls fix this:

https://www.loom.com/share/dfc42a94c1fd4f6c8973cab78e3468ef

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @StarMindz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (StarMindz) [reply src/components/Calculator.js:1]: @topeogunleye Good Morning. Can you please help out with this. I've tried everything possible. Can't find what is wrong. If you can help me point out the fault. That will be much appreciated. Thank you

Comment (topeogunleye) [reply src/components/Calculator.js:1]: > @topeogunleye Good Morning. Can you please help out with this. I've tried everything possible. Can't find what is wrong. If you can help me point out the fault. That will be much appreciated. Thank you

Hi @StarMindz, sure no problem. Please send me a direct message on slack.

**Pull request merged**